packages/web-app-files/tests/unit/views/PublicLink.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('PublicLink', () => {
+  test.todo('should have the background image set')
+  test.todo('should display the page title')
+  test.todo('should display the logo image inside login card')
+  test.todo('should display the configuration theme general slogan as login card footer')
+  describe('when "loading" is set as true', () => {
```

Comment:
In general I like phrases like `when the view is still loading` or `when the view is not loading anymore` a little bit better, because it feels more natural. Just a nitpick though, as technically you're correct here.